tests/test_search.py

```diff
@@ -0,0 +1,11 @@
+from search import create_app
+
+
+def test_empty_response() -> None:
+    app = create_app()
+    client = app.test_client()
+    response = client.get('/search')
+    assert response.status == '200 OK'
+    assert response.data == (b'<?xml version="1.0" encoding="UTF-8"?>'
+                             b'<item-list xmlns="http://libero.pub"/>')
```

Comment:
Would prefer testing the structure.

Reply:
can you provide an example please?

Reply:
Some equivalent of https://phpunit.readthedocs.io/en/7.4/assertions.html#assertxmlstringequalsxmlstring, so makes sure that two XML documents are the equal rather than two strings being the same.

Reply:
```
>>> from lxml import etree
>>> etree.fromstring(b'<?xml version="1.0" encoding="UTF-8"?><item-list xmlns="http://libero.pub"></item-list>').getchildren() == []
True
```
but the syntax may vary depending on the XML library being used.

Reply:
>So the example you have linked to is actually testing against two strings and the function is entitled `assertxmlstringequalsxmlstring` which suggests the same.

Strings of XML, so it recognises `<item-list xmlns="http://libero.pub"></item-list>` and `<item-list xmlns="http://libero.pub"/>` as equal.

Reply:
I have reimplemented the xml generation and parsing using the `lxml` library.